test/solidity/blind-auction/blindAuction.js

```diff
@@ -0,0 +1,205 @@
+/*-
+ *
+ * Hedera Smart Contracts
+ *
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+const chai = require('chai')
+const { expect } = require('chai')
+const chaiAsPromised = require("chai-as-promised")
+const { ethers } = require('hardhat')
+const Constants = require('../../constants')
+chai.use(chaiAsPromised);
+
+const sleep = (ms) => new Promise((r) => setTimeout(r, ms));
+
+const deployBlindAuctionContract = async (biddingTime, revealTime, beneficiaryAddress) => {
+    const factory = await ethers.getContractFactory(Constants.Contract.BlindAuction)
+    const contract = await factory.deploy(biddingTime, revealTime, beneficiaryAddress)
+
+    await contract.deployed()
+
+    return contract;
+}
+
+describe('Solidity Errors', function () {
+  let beneficiary, wallet1;
+  const hundred = ethers.utils.parseEther("100.0");
+  const twoHundred = ethers.utils.parseEther("200.0");
+  const fiftyGwei = ethers.utils.parseUnits('50', 'gwei');
```

Comment:
These are not meaningful in our case. Let's define them with names like 10 HBARs, 50 HBARs, and so on. Also, the calculation should be based on weibars coef and tinybar-hbar coef.
```typescript
const WEIBARS_COEF = 10_000_000_000;
const TINYBAR_TO_HBAR_COEF = 100_000_000;
const TEN_HBARS = 10 * WEIBAR_COEF * TINYBAR_TO_HBAR_COEF;
```
These coef could be found in one of the constant files, I guess.

Reply:
@natanasow done :) good sueggestion